src/index.js

```diff
@@ -30,6 +30,7 @@ ReactGA.initialize(ga, {debug: false});
 firebase.initializeApp(config);
 
 firebase.auth().onAuthStateChanged((user) => {
+  store.dispatch(userPreferencesFetchRequestAction({ uid: user.uid }));
```

Comment:
This needs to handle `user == null`

<img src="https://user-images.githubusercontent.com/930039/47716677-76c55a00-dc11-11e8-932e-6a874dcc6217.png" height="30px" />